services/net/contentmigration/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local-packages" value="/root/.nuget/packages" />
```

Comment:
Point the nuget source to the local packages so that the build works in a docker image